assets/scss/gravity-forms.scss

```diff
@@ -468,3 +468,59 @@
   color      : $color_nhsuk-black;
   padding    : nhsuk-spacing(1);
 }
+
+.gf-all-fields { // style up the all-fields summary to look like it should. This is a sleight of hand.
```

Comment:
Not sure how this is sleight of hand. Could you explain more explicitly please?

Reply:
removed the sleight comment ;)